modules/luci-base/htdocs/luci-static/resources/form.js

```diff
@@ -3938,6 +3948,9 @@ var CBIDummyValue = CBIValue.extend(/** @lends LuCI.form.DummyValue.prototype */
 		if (this.href && !((this.readonly != null) ? this.readonly : this.map.readonly))
 			outputEl.appendChild(E('a', { 'href': this.href }));
 
+		if (this.hidden)
+			outputEl.setAttribute('style', 'display: none;');
```

Comment:
Please move the `hidden` handling to the `outputEl` declaration:

```javascript
outputEl = E('div', { display: this.hidden ? 'none' : null })
```

Reply:
Done